src/rulesets/oas3/__tests__/valid-example.ts

```diff
@@ -169,7 +169,7 @@ describe('valid-example', () => {
       expect(results).toEqual([
         expect.objectContaining({
           code: 'valid-example',
-          message: `"ip_address" property format should match format "${format}"`,
+          message: `"example" property format should match format "${format}"`, // hm, ip_address is likely to be more meaningful no?
```

Comment:
@philsturgeon thoughts?

Reply:
Depends? This feels like a limitation of trying to shove everything into a simple string message. We are talking about the example, but it sure could be useful to let them know its the ip_address example that is wrong.

Reply:
Yeah, exactly.
I'd expect to receive something like `"ip_address" example property format should match format "${format}"`. Will do it this way.